includes/Modules/Thank_With_Google.php

```diff
@@ -133,6 +134,18 @@ function() {
 			}
 		);
 
+		add_filter(
+			'googlesitekit_apifetch_preload_paths',
+			function ( $paths ) {
+				return array_merge(
+					$paths,
+					array(
+						'/' . REST_Routes::REST_ROOT . '/modules/thank-with-google/data/supporter-wall-prompt',
```

Comment:
This request should technically work if TwG is active, but there isn't much point if the module isn't connected. Let's mirror the logic for showing the banner to only preload this if connected (we can just move this after the relevant guard just below this).